### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by ImVad1k** [VirgilFabre/NoNameTag#2]
[Version: 1.19.2, paper] The plugin is installed on the server and works, but nicknames are still visible

**Comment by AuracleTech** [comment]
I will check that in the coming days
Thank you for reporting the issue

**Comment by AuracleTech** [comment]
I made a simple paper server in `1.19.3 ` and everything is working perfectly as soon as I typed /reload to reload the server the plugin kicked in and all nametags disappeared.
I tried to disconnect and reconnect same thing, no name tag to be seen!

I am fairly certain another plugin of yours interfere with mine, could not tell which.

![image](https://user-images.githubusercontent.com/40531871/212614400-d518ab23-86ae-49ba-975a-8e2527ebec00.png)
![image](https://user-images.githubusercontent.com/40531871/212614107-bea3998f-c31e-4f69-8e4a-5418eaac5260.png)
![image](https://user-images.githubusercontent.com/40531871/212614033-9a7a5c70-bd4c-4121-8c11-bffb4174ca7c.png)
